api/src/utils/fetch-user-count/get-user-count-query.ts

```diff
@@ -0,0 +1,31 @@
+import type { PrimaryKey } from '@directus/types';
+import type { Knex } from 'knex';
+
+export interface GetUserCountOptions {
+	excludeIds?: PrimaryKey[];
+	excludeRoles?: PrimaryKey[];
+	includeRoles?: PrimaryKey[];
+}
+
+export function getUserCountQuery(knex: Knex, options: GetUserCountOptions) {
+	// Safety check for an empty list of includeRoles, which would otherwise return all users
+	if (options.includeRoles && options.includeRoles.length === 0) {
+		return Promise.resolve({ count: 0 });
+	}
+
+	let query = knex('directus_users').count({ count: '*' }).where('status', 'active');
```

Comment:
I believe this needs an `.as('count')`. Some database vendors will auto-name the column `count(*)` instead of `count`

Reply:
A bit late, but isn't that the alias syntax for knex if you throw in a object, where the keys are used as aliases?

Reply:
TIL! Didn't know knex did it that way (their api can be a bit.. interesting at times)